src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -159,6 +161,12 @@
  * Type is {@code java.lang.String[]}.
  * Default value is {@code After, AfterClass, Before, BeforeClass, Test}.
  * </li>
+ * <li>
+ * Property {@code requiredJavadocPhrase} - Specify the comment text pattern which qualifies a
+ * method as designed for extension. Supports multi-line regex.
+ * Type is {@code java.util.regex.Pattern}.
+ * Default value is {@code ".*"}.
```

Comment:
https://github.com/checkstyle/checkstyle/issues/4358#issuecomment-301772754
> Default should be "^This implementation " (as mentioned in our html doc).

Was this changed?

Reply:
See https://github.com/checkstyle/checkstyle/pull/8203#discussion_r418503595.  This PR is a continuation of that one.

Reply:
https://checkstyle.org/config_design.html#DesignForExtension

Reply:
I am just pointing it out. If such a change is a breaking change then I recommend splitting it into a different issue that can be marked with breaking compatibility. We should try to avoid breaking compatibility in the same issue as a new feature.

Reply:
good point.
I will not create issue on this, lets users request this if they care.